common/src/main/scala/explore/common/UserPreferencesQueries.scala

```diff
@@ -14,14 +15,16 @@ import explore.model.ResizableSection
 import explore.model.enums.PlotRange
 import explore.model.enums.TimeDisplay
 import explore.model.enums.Visible
-import explore.model.layout._
+import explore.model.layout.{_, given}
```

Comment:
I think Scala 2 style `import X._` imports givens already. Scala 3 style would be `import X.{*, given}`.

Reply:
Yes, scalafmt for some reason converts them to `{_, given}` I guess we need two lines of imports

Reply:
or maybe `OrganizeImports`